mozdownload/scraper.py

```diff
@@ -696,7 +696,14 @@ def get_build_info(self):
         self.show_matching_builds(parser.entries)
         self.builds = parser.entries
         self.build_index = len(parser.entries) - 1
-
+        
+        if self.build_number:
+          self.tmp = "build"+str(self.build_number)
+          if self.tmp not in self.builds:
+            message = 'Selected build is not available.'
+            sys.logger.info(message)
```

Comment:
When you check the log files of the failed tests on Travis and AppVeyor you can see that `sys.logger` is not available.